src/core/utils/uuid.js

```diff
@@ -0,0 +1,10 @@
+import { v4 as uuidv4 } from 'uuid';
+
+/**
+ * Generates a uuid.
+ * 
+ * @returns {String}
+ */
+export function uuid() {
```

Comment:
nit: This function name is a noun, and I think it's easier to read code when function names start with verbs. Would `generateUUID` work here instead?

Reply:
yeah we can do that.